src/db.rs

```diff
@@ -471,20 +647,6 @@ impl Agate {
     pub fn write_requests(&self, request: Vec<Request>) -> Result<()> {
```

Comment:
move these to the top impl block?

Reply:
I think stay there is enough :rofl:

Reply:
why separate impl block

Reply:
Sorry I'm confused, what separates the impl block?

Reply:
I merged two impl block for `Agate`.